rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# This is our blog posts controller, where we'll be defining the methods for CRUD functionality
 class BlogPostsController < ApplicationController
```

Comment:
What type of relationship is `<` showing for BlogPostsController and ApplicationController?